src/backup.rs

```diff
@@ -170,6 +170,55 @@ impl BackupEngine {
         Ok(())
     }
 
+    /// Restore from a specified backup
+    ///
+    /// # Arguments
+    ///
+    /// * `db_dir` - A path to the database directory
+    /// * `wal_dir` - A path to the wal directory
+    /// * `opts` - Restore options
+    /// * `backup_id` - The backup id to restore
```

Comment:
Personally I don’t think that this description of the arguments is helpful. The names of the arguments are pretty self-explanatory. Perhaps it makes sense to extend the method documentation, but I don’t have a concrete suggestion.

Reply:
replace it with a comment